front/app/modules/commercial/insights/admin/containers/Insights/Details/Inputs/index.tsx

```diff
@@ -137,7 +143,10 @@ const Inputs = ({
 
   return (
     <InputsContainer data-testid="insightsDetailsInputs">
-      {inputs.length > 0 && <Export />}
+      <Box display="flex" justifyContent="space-between" alignItems="center">
```

Comment:
This is not properly styled right now but I need to merge the PR with the tooltips first - here is what it will look like in the end.
![image](https://user-images.githubusercontent.com/25869467/140948263-304585cf-e906-4405-bdd7-08e2e012caab.png)